unyt/unit_object.py

```diff
@@ -625,8 +626,11 @@ def get_base_equivalent(self, unit_system="cgs"):
         unit_system = unit_system_registry[str(unit_system)]
         try:
             units_string = unit_system[self.dimensions]
-        except MissingMKSCurrent:
-            units_string = em_conversions[self.dimensions][0]
+        except MissingMKSUnit:
+            if self.dimensions in em_conversions:
+                units_string = em_conversions[self.dimensions][0]
+            else:
+                raise UnitsNotReducible(self, unit_system.name)
```

Comment:
If you add luminance and amount to the other unit systems, you don't need to raise this error, you could just convert the data.

Here's my suggestion, for CGS use Lambert as the default luminance unit. Otherwise use candelas (unless there's an obvious other choice for the other unit systems?). Arguing for this: we also use "radian" as the default angle unit for all the other unit systems. As far as I can tell the units for luminance [are weird](https://twitter.com/njgoldbaum/status/984156999792209920).

Is it really helpful to raise this error? If we make this one concession to keeping things more SI-like by default I think it will be easier to use the library.